### PR TITLE
Send proposed transactions

### PR DESCRIPTION
**Pull request by fluidvanadium**
lives on #1010
#1012
#1013
#1014

### CODE REVIEW

**Comment by fluidvanadium** [comment]
running CI

**Comment by fluidvanadium** [comment]
okay, this is taking ABSURDLY long. that is a bug we have gotta fix

File: Cargo.toml [hunk 1]
```diff
@@ -17,14 +17,14 @@ members = [
 resolver = "2"
 
 [workspace.dependencies]
-zcash_address = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zcash_client_backend-0.12.1-1-g11ce156b_avoid_wallet_write" } # git tag -s $(git describe --dirty)
-zcash_client_backend = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zcash_client_backend-0.12.1-1-g11ce156b_avoid_wallet_write", features = ["lightwalletd-tonic", "orchard", "transparent-inputs"] }
-zcash_encoding = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zcash_client_backend-0.12.1-1-g11ce156b_avoid_wallet_write" }
-zcash_keys = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zcash_client_backend-0.12.1-1-g11ce156b_avoid_wallet_write", features = ["orchard"] }
+zcash_address = { git = "https://github.com/zingolabs/librustzcash.git", rev = "4c67b47" }
```
Comment (zancas) [inline Cargo.toml]: I am -1 on depending on a commit hash.   Just toss a tag on it!

Comment (zancas) [reply Cargo.toml]: But maybe this is just an "in-flight" hack?

Comment (fluidvanadium) [reply Cargo.toml]: i used the toss a tag script and it did something weird

File: zingolib/src/wallet/send.rs [hunk 2]
```diff
@@ -865,13 +866,20 @@ impl LightWallet {
...
         let txid = transaction.txid();
-        if txid.to_string() != transaction_id {
-            // return Err(format!(
-            dbg!(
-                "served txid {} does not match calulated txid {}",
-                transaction_id,
-                txid,
-            );
+
+        if let Ok(serverz_txid_bytes) = dbg!(serverz_transaction_id).into_bytes().try_into() {
```
Comment (zancas) [inline zingolib/src/wallet/send.rs:870]: We need to remember to excise this before merge.

**Pull request closed without merging**